maxim/apis/maxim_apis.py

```
@@ -689,45 +694,205 @@ def get_folders(self) -> List[Folder]:
+        # Create FileVariablePayload for this entry
+        if file_attachments:
+            file_payload = FileVariablePayload(
+                text=entry.column_name,
+                files=file_attachments,
+                entryId=entry_id
+            )
+            return file_payload
+        
+        return None
 
+    def _process_attachment(self, attachment) -> tuple[bytes, str, int]:
```

Ruff (0.12.2)

Missing type annotation for function argument `attachment`

(ANN001)